### PR TITLE
Review for release notes for 0.13.3

### PR DESCRIPTION
**Pull request by determined-dsw**
## Commentary (optional)

Please just look at docs/release-notes.txt to review.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: docs/release-notes.txt [hunk 1]
```diff
@@ -6,6 +6,102 @@ Release Notes
...
+
+**Release Date:** September 8, 2020
+
+**Bug Fixes**
+
+- Fix a bug where ``det-deploy local cluster-up`` was failing.
+ 
+- Fix a bug where experiment labels were not displayed on the experiment list page.
+
+- Fix a bug with decoding API responses because of unexpected non-numeric metric values.
```
Comment (hkang1) [inline docs/release-notes.txt:20]: LGTM!

**Review by hamidzr — COMMENTED** [review]
WebUI release notes LGTM

File: docs/release-notes.txt (hunk 1 above)
Comment (determined-dsw) [reply docs/release-notes.txt:20]: will do

File: docs/release-notes.txt [hunk 2]
```diff
@@ -6,6 +6,102 @@ Release Notes
...
+Version 0.13.3
+^^^^^^^^^^^^^^
+
+**Release Date:** September 8, 2020
+
+**Bug Fixes**
+
+- Fix a bug where ``det-deploy local cluster-up`` was failing.
+ 
+- Fix a bug where experiment labels were not displayed on the experiment list page.
```
[the remark this answers is not in the document]
Comment (determined-dsw) [reply docs/release-notes.txt:18]: will do.  We need some official categories for this stuff.

**Pull request closed without merging**